src/common/components/ui/PopoverIcon.tsx

```diff
@@ -28,7 +28,10 @@ const PopoverIcon: React.FC<PopoverIconProps> = ({
       onClick={(e) => e.preventDefault()}
       className={classNames('pf-c-form__group-label-help', buttonClassName)}
     >
-      <IconComponent noVerticalAlign={noVerticalAlign} />
+      <IconComponent
+        noVerticalAlign={noVerticalAlign}
+        style={{ fill: 'var(--pf-c-form__group-label-help--Color)' }}
```

Comment:
This override is not necessary. The issue is caused by default Button `variant` prop. According to PatternFly docs, the question mark icon at the form should be gray, which resembles to the `plain` button variant.

Since the PopoverIcon is not used only for form input help popovers, as a solution I'd recommend creating a dedicated component for the help icons e.g. `FormGroupLabelHelp` which would render `PopoverIcon` with correct props (IconComponent and variant) and replace the existing usage of PopoverIcon in form fields.

Reply:
I change the compoment to set the default variant to plain and explicitly define it as link where needed.